src/sentry/conduit/auth.py

```diff
@@ -0,0 +1,96 @@
+import time
+import uuid
+from typing import NamedTuple
+
+from django.conf import settings
+
+from sentry.utils import jwt, metrics
+
+TOKEN_TTL_SEC = 600  # 10 minutes
+
+
+class ConduitCredentials(NamedTuple):
+    token: str
+    channel_id: str
+    url: str
+
+
+def generate_channel_id() -> str:
+    """
+    Generate a unique channel ID for a Conduit stream.
+
+    Returns:
+        UUID string
+    """
+    return str(uuid.uuid4())
+
+
+def generate_conduit_token(
+    org_id: int,
+    channel_id: str,
+    issuer: str | None = None,
+    audience: str | None = None,
+    conduit_private_key: str | None = None,
```

Comment:
```suggestion
    issuer: str = settings.CONDUIT_JWT_ISSUER,
    audience: str = settings.CONDUIT_JWT_AUDIENCE,
    conduit_private_key: str = settings.CONDUIT_PRIVATE_KEY,
```

Reply:
This looks cleaner but makes mocking very complicated for tests and `conduit_private_key` doesn't have a default value so it still needs to be checked for `None`